specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json

```
@@ -0,0 +1,1192 @@
+    },
+    "DetectionStatus": {
+      "description": "Status of detection results. One of CREATED, RUNNING, READY, and FAILED.",
+      "enum": [
+        "CREATED",
+        "RUNNING",
+        "READY",
+        "FAILED"
+      ],
+      "example": "READY",
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": false,
+        "name": "DetectionStatus",
+        "values": [
+          {
+            "value": "CREATED"
+          },
+          {
+            "value": "RUNNING"
+          },
+          {
+            "value": "READY"
+          },
+          {
+            "value": "FAILED"
+          }
+        ]
+      }
+    },
+    "AnomalyState": {
+      "required": [
+        "timestamp"
+      ],
+      "properties": {
+        "timestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The timestamp for this anomaly.",
+          "example": "2022-01-01T00:00:00Z"
+        },
+        "value": {
+          "$ref": "#/definitions/AnomalyValue"
+        },
+        "errors": {
+          "type": "array",
+          "description": "Error message for the current timestamp.",
+          "items": {
+            "$ref": "#/definitions/ErrorResponse"
+          }
+        }
+      },
+      "type": "object"
+    },
+    "AnomalyValue": {
+      "required": [
+        "isAnomaly",
+        "score",
+        "severity"
+      ],
+      "properties": {
+        "isAnomaly": {
+          "type": "boolean",
+          "description": "True if an anomaly is detected at the current timestamp.",
+          "example": true
+        },
+        "severity": {
+          "type": "number",
+          "description": "Indicates the significance of the anomaly. The higher the severity, the more significant the anomaly is.",
+          "example": 0.8,
+          "minimum": 0.0,
+          "maximum": 1.0
+        },
+        "score": {
+          "type": "number",
+          "description": "Raw anomaly score of severity, will help indicate the degree of abnormality as well.",
+          "example": 0.3,
+          "minimum": 0.0,
+          "maximum": 2.0
+        },
+        "interpretation": {
+          "type": "array",
+          "items": {
+            "description": "Interpretation contains more details of the anomaly, which will help with root cause analysis.",
+            "allOf": [
+              {
+                "$ref": "#/definitions/AnomalyInterpretation"
+              }
+            ]
+          }
+        }
+      },
+      "type": "object"
+    },
+    "AnomalyInterpretation": {
+      "properties": {
+        "variable": {
+          "type": "string",
+          "description": "Variable.",
+          "example": "variable_1"
+        },
+        "contributionScore": {
+          "type": "number",
+          "description": "This score shows the percentage contributing to the anomalous timestamp. A number between 0 and 1.",
+          "example": [
+            0.3324159383
+          ]
```

`contributionScore` is `type: number` so the example must be a number and not an array.
```suggestion
          "example": 0.3324159383
```